packages/adminrouter/extra/src/lib/cache.lua

```diff
@@ -234,8 +234,24 @@ local function fetch_and_store_marathon_apps(auth_token)
           goto continue
        end
 
+       local do_rewrite_requrl = labels["DCOS_SERVICE_REWRITE_REQUEST_URLS"]
+       if do_rewrite_requrl == nil then
```

Comment:
I think that this block is not necessary if you change `elseif` to `if` on L241.

Reply:
I wanted to be explicit, but yeah you are right. Thanks!

Reply:
Yep, you can put a comment if you like.